src/app/index.html

```diff
@@ -90,6 +92,7 @@
       window.addEventListener("load", function() {
 
         System.import('app/boot').then(null, console.error.bind(console));
+        System.import('dompurify').then(null, console.error.bind(console));
```

Comment:
@youdz : I think this was left in from an early attempt to get this working. I've removed that line and everything still works fine. App runs. Tests run...